src/apps/components/AppDetailsPage/DataPrivacyCard.tsx

```diff
@@ -1,47 +1,51 @@
-// @ts-strict-ignore
-import CardTitle from "@dashboard/components/CardTitle";
 import ExternalLink from "@dashboard/components/ExternalLink";
 import Skeleton from "@dashboard/components/Skeleton";
-import { Card, CardContent } from "@material-ui/core";
+import { Box, BoxProps, Text } from "@saleor/macaw-ui/next";
 import React from "react";
 import { FormattedMessage, useIntl } from "react-intl";
 
 import messages from "./messages";
-import { useStyles } from "./styles";
 
-interface DataPrivacyCardProps {
+type DataPrivacyCardProps = {
   dataPrivacyUrl?: string | null;
   loading: boolean;
-}
+} & BoxProps;
 
-const DataPrivacyCard: React.FC<DataPrivacyCardProps> = ({
+export const DataPrivacyCard: React.FC<DataPrivacyCardProps> = ({
   dataPrivacyUrl,
   loading,
+  ...boxProps
 }) => {
-  const classes = useStyles();
   const intl = useIntl();
 
   if (!dataPrivacyUrl && !loading) {
     return null;
   }
 
+  const renderContent = () => {
+    if (loading) {
+      return <Skeleton />;
+    }
+
+    if (dataPrivacyUrl) {
+      return (
+        <ExternalLink href={dataPrivacyUrl} target="_blank">
+          <FormattedMessage {...messages.dataPrivacyDescription} />
+        </ExternalLink>
+      );
+    } else {
+      return <Text>{intl.formatMessage(messages.noDataPrivacyPage)}</Text>;
+    }
+
+    throw new Error("Leaking switch statement");
```

Comment:
```suggestion
    
    if (!dataPrivacyUrl) {
      return <Text>{intl.formatMessage(messages.noDataPrivacyPage)}</Text>
    }
    
    return (
      <ExternalLink href={dataPrivacyUrl} target="_blank">
        <FormattedMessage {...messages.dataPrivacyDescription} />
      </ExternalLink>
    );
```

Was this error necessary? Seems strange 🤔

Reply:
Yes. Component should render only for cases I want it to. If none of conditions are met, this is a bug in the code. Should not happen. Just like leaky switch/cases. Hence throwing error to quickly see that something wrong happened. If I return "null" here, we will never now if there is a bug

Reply:
That's if/else with returning statements inside. Exception throw is unreachable

Reply:
Fixed